projects/plugins/boost/readme.txt

```diff
@@ -165,25 +165,19 @@ If you run into compatibility issues, please do let us know. You can drop us a l
 1. Manage your Jetpack Boost settings
 
 == Changelog ==
-### 1.3.0-beta - 2021-10-04
-#### Security
-- Critical CSS: Add permissions checks to AJAX endpoints used when dismissing Critical CSS Recommendations.
-
+### 1.4.1-beta - 2022-03-08
 #### Added
-- Critical CSS: Add extra information to "fetch" errors when generating Critical CSS.
-- Critical CSS: Added explanation for mod-security HTTP 418 errors.
-- Critical CSS: Added stats tracking for generation outcomes.
-- Critical CSS: Added step-by-step instructions for Advanced Recommendations.
-- Critical CSS: More descriptive error message if critical css is failing because of x-frame-options deny config.
-- Speed Scores: Added "without Boost" speed score indicator.
+- Added internal support for Cloud Critical CSS.
+- Added My Jetpack page.
```

Comment:
This is not true. My jetpack was added in 1.4.0 but it is kept hidden unless a constant is specified.

Reply:
Thanks Adnan e22bc6b31c